File: 1.C/2.c
/*16. Escreva um programa que leia 10 números e:
a. Escreva o MENOR valor lido e o MAIOR valor lido
b. Some-os.
c. Imprima sua média*/

#include <stdio.h>
#include <locale.h>

int main()
{
    setlocale(LC_ALL, "Portuguese");

    int CUSTOM, i, SOMA = 0, maior, menor;
    float MEDIA;

    printf("Quantos números? ");
    scanf("%d", &CUSTOM);

    int ENTRADA[CUSTOM];

    for (i = 0; i < CUSTOM; i++)
    {
        printf("%d. Número: ", i + 1);
        scanf("%d", &ENTRADA[i]);

        SOMA += ENTRADA[i];

        if (i == 0)
        {
            maior = ENTRADA[i];
            menor = ENTRADA[i];
        }
        else
        {
            if (ENTRADA[i] > maior)
                maior = ENTRADA[i];
            if (ENTRADA[i] < menor)
                menor = ENTRADA[i];
        }
    }

    MEDIA = (float)SOMA / CUSTOM;

    printf("MENOR valor: %d\n", menor);
    printf("MAIOR valor: %d\n", maior);
    printf("SOMA: %d\n", SOMA);
    printf("MÉDIA: %.2f\n", MEDIA);

    return 0;
}

File: 2.JS/2.js
/*25. Escreva um programa que leia 10 números e:
a. Escreva o MENOR valor lido e o MAIOR valor lido
b. Some-os.
c. Imprima sua média*/

let custom = 10;
let entrada = [2, 4, 6, 8, 10, 3, 6, 5, 11, 30];
let soma = 0;
let media;

for (let i = 0; i < custom; i++) {
    soma += entrada[i];
}

media = soma / custom;

console.log("SOMA: " + soma);
console.log("MÉDIA: " + media.toFixed(2));
